### PR TITLE
Add fix:* tags on various analysers (see #1182)

### PR DESCRIPTION
**Pull request by PanierAvide**
Added fix:* tags which seemed logical according to the kind of analyser.

### CODE REVIEW

**Comment by frodrigo** [comment]
Thank you, it is a lot of work.
Please can you clarify the meaning of `fix:picture`?

**Comment by PanierAvide** [comment]
`fix;picture` is for when this can be solved using street-level pictures (mapillary, openstreetcam)

File: analysers/analyser_merge_traffic_signs.py [hunk 1]
```diff
@@ -79,7 +79,7 @@ def __init__(self, config, error_file, logger, item, classs, level, tags, otype,
                 ['`{}={}`'.format(kv[0], kv[1] if kv[1] else '*') for kv in selection.items()]
             ))
 
-        self.def_class_missing_official(item = item, id = classs, level = level, tags = ['merge'] + tags,
+        self.def_class_missing_official(item = item, id = classs, level = level, tags = ['merge', 'highway', 'fix:picture'] + tags,
```
Comment (Marc-marc-marc) [inline analysers/analyser_merge_traffic_signs.py:82]: also fix:survey

File: analysers/analyser_merge_water_FR.py [hunk 2]
```diff
@@ -119,7 +119,7 @@ def __init__(self, config, logger = None):
 class SubAnalyser_Merge_Water_FR(SubAnalyser_Merge_Dynamic):
     def __init__(self, config, error_file, logger, item, id, level, title, select, tags, generate_tags):
         SubAnalyser_Merge_Dynamic.__init__(self, config, error_file, logger)
-        self.def_class_missing_official(item = item, id = id, level = level, tags = ['merge'], title = title)
+        self.def_class_missing_official(item = item, id = id, level = level, tags = ['merge', 'water', 'fix:survey', 'fix:imagery'], title = title)
```
Comment (Marc-marc-marc) [inline analysers/analyser_merge_water_FR.py:122]: fountain also with fix:chair

File: analysers/analyser_merge_street_objects.py [hunk 3]
```diff
@@ -43,7 +43,7 @@ def __init__(self, config, logger = None):
 class SubAnalyser_Merge_Street_Objects(SubAnalyser_Merge_Dynamic):
     def __init__(self, config, error_file, logger, classs, level, otype, conflation, title, object, selectTags, generateTags, mapping, source, layer):
         SubAnalyser_Merge_Dynamic.__init__(self, config, error_file, logger)
-        self.def_class_missing_official(item = 8360, id = classs, level = level, tags = ['merge', 'leisure'],
+        self.def_class_missing_official(item = 8360, id = classs, level = level, tags = ['merge', 'leisure', 'fix:survey', 'fix:picture'],
```
Comment (Marc-marc-marc) [inline analysers/analyser_merge_street_objects.py:46]: osmose produce a merge, so mapillary found it on a street level imagery. so imho fix:chair is also doable

**Review by Marc-marc-marc — APPROVED** [review]
I also don't see the difference between fix:picture and fix:survey (depending on the pictures, one picture is or isn't enough to find the information. this makes it impossible for me to say "a survey is always needed and not picture").
but I have nothing against this addition, the whole PR is anyway a big improvement!but it doesn't bother me to see it added, the whole thing is a great improvement over the current situation anyway.

I have posted a few comments, but which do not deserve to block the adoption of this PR

**Comment by PanierAvide** [comment]
Thanks for your changes suggestions, I will integrate them soon.

Please read here [my comment here](https://github.com/osm-fr/osmose-backend/issues/1182#issuecomment-780429801) about how I see `fix:picture` and `fix:survey` differently. There is no 1-1 match between street-level pictures and aerial imagery, and ground survey might not always be possible (think of motorways, where "manual" survey is quite impossible whereas taking pictures is easy). As a data reuser and developer, I see this as an interesting tag to put on issues. As it doesn't replace any existing feature, it's just another information. Everyone would be free to use or not use it.

**Comment by PanierAvide** [comment]
Let's take another example : if we had an analyser about tree diameters (centimeter precision), it would be fixable by survey. But no way to check that info using a picture or imagery. All `fix:picture` will be associated to `fix:survey`, but not all `fix:survey` could be solved using pictures.

File: analysers/analyser_merge_water_FR.py (hunk 2 above)
Comment (PanierAvide) [reply analysers/analyser_merge_water_FR.py:122]: I'm not sure if `fix:chair` makes sense here, you will either need an imagery or to go on ground to integrate these features. Isn't `fix:chair` about things you can fix by just checking Osmose information (refs, topology issues, knowledge from your brain...) ?

File: analysers/analyser_merge_street_objects.py (hunk 3 above)
Comment (PanierAvide) [reply analysers/analyser_merge_street_objects.py:46]: Same as above

File: analysers/analyser_merge_traffic_signs.py (hunk 1 above)
Comment (PanierAvide) [reply analysers/analyser_merge_traffic_signs.py]: Added, thanks :+1:

**Comment by frodrigo** [comment]
Thank you a lot for all this work.

**Pull request merged**